src/components/applications/kanban/Consents.tsx

```diff
@@ -134,7 +146,7 @@ const Consents: React.FC<ConsentsProps> = ({ handleClose, dialog }) => {
         </DialogTitle>
         <DialogContent>
           <DialogContentText id="reset-form-contents" color="textPrimary">
-            Are you sure you want to revoke consent for UNID {deleteUnid}?
+            Are you sure you want to revoke consent for user {username} with scopes {scope}?
```

Comment:
Will leave this with @rwilkins108 to confirm the dialog text. Should we include the app name here, or leave it as it is?

Reply:
Yes, I think the app name should be here as well. @dev-patmazo @ddsoriano26

Reply:
Application name were included.